game/webserver.cc

```diff
@@ -1,22 +1,21 @@
 #include "webserver.hh"
 #include "game.hh"
 
-#ifdef USE_WEBSERVER
 #include <boost/asio.hpp>
```

Comment:
Hmm this should be installed and be listed within the vcpkg.json
-> `boost-asio`

if cppnetlib has a method to get the IP address we can also get rid of this include / dependency

Reply:
Confirmed: Adding `boost-asio` or an equivalent to vcpkg.json will fix the build.

However i do get weird errors regarding caching on windows:
```
songs/error: Could not save C:\Users\Arjan\AppData\Roaming\performous\cache\songs.json: [json.exception.type_error.316] invalid UTF-8 byte at index 45: 0x72    
```